relay-dynamic-config/src/defaults.rs

```diff
@@ -187,6 +187,7 @@ pub fn add_span_metrics(project_config: &mut ProjectConfig) {
                 ("", "ttfd"),
                 ("", "ttid"),
                 ("span.", "main_thread"),
+                ("", "os.name"),
```

Comment:
Do we need this tag on both `exclusive_time` and `exclusive_time_light`?

Reply:
Good question! not right now, but would be nice to have for some of the app.start* spans that we don't plan to facet by transaction in the app startup module. 

Related to this however, the `ttid` and `ttfd` tag will never need `exclusive_time_light` since the tags will only ever be used in the context of a "transaction".